src/mame/sinclair/atm.h

```diff
@@ -0,0 +1,106 @@
+// license:BSD-3-Clause
+#ifndef MAME_INCLUDES_ATM_H
+#define MAME_INCLUDES_ATM_H
```

Comment:
This should match the new project structure – it should be changed to `MAME_SINCLAIR_ATM_H`.  Existing headers should be changed as well, but it’s going to take some time for them to all be caught.

Reply:
What are the rules for prefix? Under /sinclair I see mix of MAME_MACHINE and MAME_INCLUDES